net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -17,14 +17,21 @@ start() {
 
 		exit 1
 	fi
+	config_load ampr-ripd
+	config_get enabled network enabled 0
+	if [ "$enabled" != 1 ]; then
+		echo "ampr-ripd disabled via config"
+		exit 1
+	fi
 	if [ ! -d /var/lib/ampr-ripd ]; then
```

Comment:
By the way, it's kind of redundant to check if a directory exists before doing `mkdir -p`.